platform/admin/ui/src/main/webapp/templates/installer/configuration.handlebars

```diff
@@ -11,16 +17,3 @@
  *
  **/
  --}}
-<h3>Additional System Settings</h3>
-<form id="config-form" class="inner-scroll scroll-area ps-container" method="GET">
-    <div class="well white">
-        <p class="lead">
-            <br/>
-            After installation is complete there are additional configurations that can be made
-            to change hostname, ports and certificates. A system shutdown is required in order to make these changes.<br/><br/>
-
-            Global system properties can be configured in the INSTALL_HOME/etc/system.properties file and certificates for the host can be added to the systemKeystore/systemTruststore in INSTALL_HOME/etc/keystore/<br/>
-
```

Comment:
I think we still need all this text in here still to let admins know that they can use the system.properties file to make additional changes.